### PR TITLE
Split apply

### PR DESCRIPTION
**Pull request by Raffo**
Part 1 of the splitting issue. This still misses doing the apply in `cluster.py` to maintain backwards compatibility.

### CODE REVIEW

**Comment by mikkeloscar** [comment]
Rebased on master.

File: cluster/manifests/secretary/deployment.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
         args:
         - /meta/credentials
         - --application-id=secretary
-        - --mint-bucket=s3://{{.ConfigItems.MINT_BUCKET}}
+        - --mint-bucket=s3://{{.ConfigItems.gerry_mint_bucket}}
```
Comment (mikkeloscar) [inline cluster/manifests/secretary/deployment.yaml]: I would remove the `gerry_` prefix here, it's just a mint bucket. If it's for the case of several different mint buckets then I would prefix it with the application name e.i `secretary`.

What do you think @hjacobs and @linki ?

File: cluster/manifests/logging-agent/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+          mountPath: /mnt/scalyr-config
+          readOnly: false
+
+      - name: scalyr-agent
+        image: registry.opensource.zalan.do/eagleeye/scalyr-agent:0.1
+
+        env:
+        # Note: added for scalyr-config-base, but not needed by the scalyr-agent itself.
+        - name: WATCHER_SCALYR_API_KEY
+          value: "{{ .ConfigItems.SCALYR_ACCESS_KEY }}"
```
Comment (hjacobs) [inline cluster/manifests/logging-agent/daemonset.yaml]: this should be `ConfigItems.scalyr_access_key` (lowercase)

Comment (mikkeloscar) [inline cluster/manifests/logging-agent/daemonset.yaml]: fixed.

**Comment by hjacobs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**